skll/config.py

```diff
@@ -606,7 +607,7 @@ def _parse_config_file(config_path, log_level=logging.INFO):
                                                     "sampler_parameters"))
     fixed_sampler_parameters = yaml.safe_load(fixed_sampler_parameters)
     param_grid_list = yaml.safe_load(_fix_json(config.get("Tuning", "param_grids")))
-    pos_label_str = config.get("Tuning", "pos_label_str")
+    pos_label_str = safe_float(config.get("Tuning", "pos_label_str"))
```

Comment:
No, because the [default value](https://github.com/EducationalTestingService/skll/blob/master/skll/config.py#L85) in config.py for `pos_label_str` is the empty string `''`. So, the value will still be the empty string which basically gets happily ignored when doing the sorting in `Learner._create_label_dict()`. But you are right in that we should make the default values of `pos_label_str` be `None` both via the API and the configuration file. I will make a PR for this.